gldemo/plugin.cpp

```diff
@@ -185,12 +196,16 @@ class gldemo : public threadloop {
 			glFlush();
```

Comment:
I did a little bit of reading into this. Apparently many modern drivers completely ignore `glFlush()` entirely. We really should be using a separate mechanism to synchronize accesses to these FBOs, like so:

https://www.khronos.org/opengl/wiki/GLAPI/glCheckFramebufferStatus

` GLenum glCheckFramebufferStatus(GLenum target​);`

However, if we don't use `glFinish` at all, we'll probably start messing up our timing, because otherwise the CPU thread doesn't really know when the rendering is "done". I'm going to switch it to `glFinish()` and test. However, I'm worried that using `glFinish()` will actually stall more than just the `gldemo` thread... will require some evaluation.